api/src/main/resources/org/openmrs/api/db/hibernate/ConceptMap.hbm.xml

```diff
@@ -19,7 +19,7 @@
  
 		<id name="conceptMapId" type="java.lang.Integer" column="concept_map_id" unsaved-value="0">
 			<generator class="native">
-				<param name="sequence">concept_map_concept_map_id_seq</param>
```

Comment:
According to the convention, `concept_map_concept_map_id_seq` looks like the correct one. Why do you think `concept_reference_map_concept_map_id_seq` is the correct one?

Reply:
If you look at the table name [here](https://github.com/openmrs/openmrs-core/blob/e8c7cb456f34561c5714f40ee20342230e6357a3/api/src/main/resources/org/openmrs/api/db/hibernate/ConceptMap.hbm.xml#L18) it is concept_reference_map and id is concept_map_id. Hence the created sequence is **concept_reference_map_concept_map_id_seq** . I have cross checked this in PostgreSQL created sequences.

Reply:
I have attached the screenshot in ticket here https://issues.openmrs.org/browse/TRUNK-5800

Reply:
Does this have any effect to existing installations which may already be using `concept_map_concept_map_id_seq`?

Reply:
For MySQL there should not be any effect since it uses auto increment for id. For DB's that make use of table for a sequence as in case of PostgreSQL, this sequence does not get detected as it does not exists. For DB's making use of Sequence as a table we need to properly map them via hibernate.